lib/WTSI/NPG/HTS/PacBio/RunMonitor.pm

```diff
@@ -202,6 +207,13 @@ sub _run_info {
                  $job->{IndexOfLook};
 }
 
+sub _build_dest_collection  {
+  my ($self) = @_;
+
+  return catdir($DEFAULT_ROOT_COLL, $self->run_name);
```

Comment:
from npg_pacbio_runmonitor.pl  - Can't locate object method "run_name" via package "WTSI::NPG::HTS::PacBio::RunMonitor"  (leave to RunPublisher ?)